lib/timeline/component/ItemSet.js

```diff
@@ -1225,8 +1225,8 @@ ItemSet.prototype._onDragStart = function (event) {
     }
 
     // override options.editable
-    if ((item.editable === false || !item.editable)
-    && !this.options.editable.overrideItems) {
+    if (!item.editable.updateTime && !item.editable.updateGroup
```

Comment:
This predicate is in the `ItemSet._onDragStart`, my understanding is that  `remove` is only concerned with `Item._repaintDeleteButton`. Shouldn't this predicate not be affected by the value of `remove`?